cmd/build/command.go

```diff
@@ -190,7 +190,11 @@ func (bc *Command) getBuilder(options *types.BuildOptions, okCtx *okteto.Context
 			}
 			builder = buildv2.NewBuilder(bc.Builder, bc.Registry, bc.ioCtrl, okCtx, bc.k8slogger, callbacks)
 		} else {
-			builder = buildv1.NewBuilder(bc.Builder, bc.ioCtrl)
+			return nil, oktetoErrors.UserError{
+				E: fmt.Errorf("specified Okteto Manifest doesn't contain a build section"),
```

Comment:
```suggestion
				E: fmt.Errorf("Specified Okteto Manifest doesn't contain a build section"),
```

nit

Reply:
Errors should not be capitalised. When printing them we capitalise the first letter to show it to the user